libafl/src/schedulers/queue.rs

```diff
@@ -55,6 +58,12 @@ where
                 .map(|id| state.corpus().next(id))
                 .flatten()
                 .unwrap_or_else(|| state.corpus().first().unwrap());
+
+            self.runs_in_current_cycle += 1;
+            // TODO deal with corpus_counts decreasing due to removals
```

Comment:
Can we already merge this anyway?

cc @tokatoka

Reply:
IIRC yes since the Power & Weighted both handle it this way. It's still TODO there

Reply:
👍